osu.Game.Resources/osu.Game.Resources.csproj

```diff
@@ -14,4 +14,7 @@
     <EmbeddedResource Include="Textures\**\*" />
     <EmbeddedResource Include="Tracks\**\*" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
not sure this is necessary

Reply:
isn't necessary, just checked
I'll remove it in a second